shared/js/ui/views/privacy-practices.es6.js

```diff
@@ -0,0 +1,44 @@
+const ParentSlidingSubview = require('./sliding-subview.es6.js')
+const SiteModel = require('./../models/site.es6.js')
+const overviewTemplate = require('./../templates/shared/privacy-practices-overview.es6.js')
+const detailsTemplate = require('./../templates/shared/privacy-practices-details.es6.js')
+
+function PrivacyPractices (ops) {
+    this.model = null
+    this.currentModelName = null
+    this.template = ops.template
+
+    ParentSlidingSubview.call(this, ops)
+
+    this._cacheElems('.js-privacy-practices', [
+        'overview',
+        'details',
+    ])
+
+    this.setupClose();
+    this.renderAsyncContent();
+}
+
+PrivacyPractices.prototype = $.extend({},
+    ParentSlidingSubview.prototype,
+    {
+        renderAsyncContent: function () {
+            const random = Math.round(Math.random()*100000)
+            this.currentModelName = 'site' + random
```

Comment:
I copied this from the `GradeDetails` view. The prospect of collisions worries me, this is bound to cause a problem for someone at some point.

We've got a bit of code in the core code that generates a UUID for each model as it's created. Should we port this to here?

Reply:
As I was working on https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/7 I realized I can just pass down the global site model here. I'll rejig this.

Reply:
Yes, inheriting the one Site model would be preferred, but the other PR is still appreciated and will be merged.

Reply:
Inheriting the Site model is preferred to init'ing a new one here. The other PR is needed though so that will still get merged obvi.